src/assets/css/global/_main.scss

```diff
@@ -395,13 +395,18 @@ hr {
       }
     }
 
+    input {
+      text-align: right;
+    }
+
     .mat-form-field-infix {
       width: 100%;
     }
 
     .mat-form-field-outline-start,
     .mat-form-field-outline-end {
-      border-radius: 0;
+      border-bottom-right-radius: 0;
+      border-top-right-radius: 0;
```

Comment:
Check if excluding `.mat-form-field-outline-start` will not solve it too. Then you could keep `border-radius: 0;` as the only rule.